### PR TITLE
fix: Query lease periods per slot; use exact int types

### PR DESCRIPTION
**Pull request by emostov**
Changes:

- Dynamically `leasePeriodsPerSlot`, but use a fallback to ensure compatibility w/ current `rococo-v1` branch which does not expose the constant in metadata (https://github.com/paritytech/polkadot/pull/2862)
- Use exact int type annotations for relevant queries instead of `AbstractInt`

### CODE REVIEW

File: src/services/paras/ParasService.ts
Comment (TarikGul) [inline src/services/paras/ParasService.ts:265]: Assuming this will be `8` in the future, and there will be no need for `LEASE_PERIODS_PER_SLOT_FALLBACK`?

Comment (emostov) [reply src/services/paras/ParasService.ts:265]: Yup exactly, check line 35

File: src/services/paras/ParasService.ts
Comment (TarikGul) [inline src/services/paras/ParasService.ts:235]: Good catch!

**Review by TarikGul — APPROVED** [review]
LGTM, after this is merged I will pull this into the test branch and adjust it appropriately.

**Pull request merged**
